packages/vite/src/node/server/index.ts

```diff
@@ -120,6 +120,12 @@ export interface ServerOptions {
    * Should start and end with the `/` character
    */
   base?: string
+  /**
+   * Restrict files that could be served via '/\@fs/'.
+   *
+   * Default to the project root (`config.root`).
+   */
+  fsServeRoot?: string
```

Comment:
**though:** Can we document this a bit more? What is it? A path? A regexp pattern?
Do we need to add it to the [docs](https://github.com/vitejs/vite/tree/main/docs)?